### PR TITLE
Media Complication

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Add a media complication.

https://user-images.githubusercontent.com/231923/178437666-f9f1eab7-a29d-475f-98b9-a1378420ae77.mp4

#### WHY

Show example of live updating complication focused on media.

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: media-sample/src/main/java/com/google/android/horologist/mediasample/complication/MediaStatusComplicationService.kt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    override fun onCreate() {
+        super.onCreate()
+
+        ComplicationServiceContainer.inject(this)
+    }
+
+    override fun previewData(type: ComplicationType): Data = renderer.previewData()
+
+    override suspend fun data(request: ComplicationRequest): Data {
+        println("Data")
```
Comment (luizgrp) [inline media-sample/src/main/java/com/google/android/horologist/mediasample/complication/MediaStatusComplicationService.kt:57]: nit: intend to merge this?

File: media-sample/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -32,4 +32,6 @@
     <string name="horologist_show_artwork_on_chip">Show Artwork on Chips</string>
     <string name="horologist_sample_playlists">Playlists</string>
     <string name="horologist_show_time_text_info">Show TimeText Info</string>
+    <string name="horologist_preview_song">6 Underground</string>
+    <string name="horologist_preview_artist">Sneaker Pimps</string>
```
Comment (luizgrp) [inline media-sample/src/main/res/values/strings.xml:36]: nit: are these being used?

File: media3-backend/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -16,4 +16,6 @@
 
 <resources>
     <string name="horologist_cant_play_item">Can\'t play item</string>
+    <string name="horologist_sample_name">Sample App</string>
+    <string name="horologist_play">Play</string>
```
Comment (luizgrp) [inline media3-backend/src/main/res/values/strings.xml:20]: nit: are these being used?

**Review by luizgrp — APPROVED** [review]

**Comment by ataulm** [comment]
Why does this sample exist here instead of WearOsSamples? (I'm sure it's demonstrating something in Horologist but it's not clear from the PR description).

**Pull request merged**